justfile

```diff
@@ -189,6 +192,14 @@ all: services gen native run
 # Run everything at once, tailored for iOS development (rebuilds iOS)
 all-ios: services gen ios run
 
+testnet-apk:
+    #!/usr/bin/env bash
+    BUILD_NAME=$(yq -r .version {{pubspec}})
+    BUILD_NUMBER=$(git rev-list HEAD --count)
+    echo $BUILD_NAME
```

Comment:
nit: whenever you are quoting bash variables, it's a good practice to use `${BUILD_NAME}` - otherwise things might be parsed improperly (e.g. if there is a space or something).